OSDOCS-7446 Add Upgrade Cautions to the release notes section (#71889)
openshift/openshift-docs: networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

* link:https://access.redhat.com/errata/RHEA-2023:1954[RHEA-2023:1954 Release of AWS Load Balancer Operator on OperatorHub Enhancement Advisory Update]

[IMPORTANT]
====

> AWS Load Balancer Operator version 1 cannot upgrade automatically from version 0 (the version available for tech preview).

Reviewer: ```suggestion
The AWS Load Balancer Operator version 1.0.0 cannot upgrade automatically from version 0.x.x, the Technology Preview version.
```

Reviewer: Can the user manually upgrade the Operator? 

> The AWS Load Balancer Operator version 1.0.0 cannot upgrade from version 0.x.x.

Author: ty

> The AWS Load Balancer Operator version 1.x.x cannot upgrade automatically from version 0.x.x, the Technology Preview version.

Reviewer 2: Should there be an (ALB) after AWS Load Balancer since the appreciation is used in the second sentence?
```suggestion
The AWS Load Balancer (ALB) Operator version 1.x.x cannot upgrade automatically from version 0.x.x, the Technology Preview version.
```

Author: fixed!

> The AWS Load Balancer Operator version 1.x.x cannot upgrade automatically from version 0.x.x, the Technology Preview version.

Reviewer 2: This is really just a style choice because I don't think you really need to say version twice, but you could say something like:
```suggestion
The AWS Load Balancer Operator version 1.x.x cannot upgrade automatically from the Technology Preview version 0.x.x.
```

Author: Of course!

> To upgrade from an earlier version, you must uninstall operand and delete namespace `aws-load-balancer-operator`.

Reviewer: ```suggestion
To upgrade from an earlier version, you must uninstall the ALB operands and delete `aws-load-balancer-operator` namespace.
```

Author: ty

> To upgrade from an earlier version, you must uninstall the ALB operands and delete `aws-load-balancer-operator` namespace.

Reviewer 2: ```suggestion
To upgrade from an earlier version, you must uninstall the ALB operands and delete the `aws-load-balancer-operator` namespace.
```

Author: ty
